snapcraft/integrations/travis.py

```diff
@@ -146,12 +100,98 @@ def enable():
     logger.info(
         'Encrypting authorization for Travis and adjusting project to '
         'automatically decrypt and use it during "after_success".')
-
-    os.makedirs(os.path.dirname(LOCAL_CONFIG_FILENAME), exist_ok=True)
     with tempfile.NamedTemporaryFile(mode='w') as fd:
         store.conf.parser.write(fd)
         fd.flush()
-        _encrypt_config(fd.name)
+        os.makedirs(
+            os.path.dirname(LOCAL_CONFIG_FILENAME), exist_ok=True)
```

Comment:
why is this in the `NamedTemporaryFile` context, I fail to see the reason for a move.